scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -2302,8 +2364,11 @@ then
 
         if [[ $errcode -ne 0 ]]; then
             wsrep_log_error "******************* FATAL ERROR ********************** "
-            wsrep_log_error "Move failed, keeping ${DATA} for further diagnosis" \
+            wsrep_log_error "Move failed."
+            if [[ -n "$WSREP_LOG_DEBUG" ]]; then
+              wsrep_log_error "Keeping ${DATA} for further diagnosis" \
```

Comment:
Can you make these two error reports use the same format?  I think the wording in the move error is `better.`
I think it would also be good to mention that the ${DATA} can be removed.
So something like:
Keeping ${DATA} for further diagnosis
Check ${DATA}/XXX.log for details
${DATA} may be removed if not needed for further diagnosis.